.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [8.0, 8.1]
+        php: [8.0, 8.1, 8.2]
```

Comment:
@nikophil, did I miss a set of include permutations below in #359? Or, how should we handle?

Reply:
Maybe just?
```diff
- - {php: 8.1, symfony: 6.1.*, use-orm: 1, use-odm: 0, use-dama: 0, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 1, use-odm: 0, use-dama: 0, deps: highest}
- - {php: 8.1, symfony: 6.1.*, use-orm: 1, use-odm: 1, use-dama: 0, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 1, use-odm: 1, use-dama: 0, deps: highest}
- - {php: 8.1, symfony: 6.1.*, use-orm: 1, use-odm: 0, use-dama: 1, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 1, use-odm: 0, use-dama: 1, deps: highest}
- - {php: 8.1, symfony: 6.1.*, use-orm: 0, use-odm: 1, use-dama: 0, deps: highest}
+ - {php: 8.2, symfony: 6.2.*, use-orm: 0, use-odm: 1, use-dama: 0, deps: highest}
```

Reply:
yes, this way the highest and the lowest [php, sf, deps] are tested with all permutations
while all others are just tested with dama:1 orm:1 odm:1